src/features/react-virtualized-renderers/fileNameCellRenderer.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import { FormattedMessage } from 'react-intl';
+import { getFileExtension } from '../../utils/file';
+import commonMessages from '../../common/messages';
+import FileIcon from '../../icons/file-icon';
+import Link from '../../components/link/Link';
+
+import baseCellRenderer from './baseCellRenderer';
+
+import type { FileNameCellRendererCellData, FileNameCellRendererParams } from './flowTypes';
```

Comment:
These types appear to only be used in this component. Do they need to be exposed for consumption via the flowTypes file?

Reply:
If a consumer imports the renderer and pass it around through props or function params, it also needs to import the type, right?